webapp/lib/view.dart

```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
 
 class ContentView {
   DivElement contentElement;
+  DashboardView dashboardView;
 
   ContentView() {
-    contentElement = new DivElement();
+    contentElement = new DivElement()..classes.add('content');
+    dashboardView = new DashboardView();
+    dashboardView.activePackages.addAll(
+      [
+        new ActivePackagesViewPartial('Urgent conversations'),
+        new ActivePackagesViewPartial('Open conversations'),
+        new ActivePackagesViewPartial('Batch replies (Week 12)'),
+      ]);
+    dashboardView.availablepackages.addAll(
+      [
+        new AvailablePackagesViewPartial('Quick Poll',
+          'Ask a question with fixed answers',
+          ['Needs: Q/A, Labelling team, Safeguarding response', 'Produces: Dashboard for distribution of answers']),
+        new AvailablePackagesViewPartial('Information Service',
+          'Answer people\'s questions',
+          ['Needs: Response protocol, Labelling team, Safeguarding response', 'Produces: Thematic distribution, work rate tracker']),
+        new AvailablePackagesViewPartial('Bulk Message',
+          'Send set of people a once off message',
+          ['Needs: Definition of who. Safeguarding response', 'Produces: Success/Fail tracker'])
+      ]);
+
+      dashboardView.pupolateActivePackages();
```

Comment:
typo nit: should be `populateActivePackages`

Reply:
Oops. Sorry for the spelling mistakes. Renamed it to render as per your other suggestion.